gotham/src/lib.rs

```diff
@@ -3,8 +3,8 @@
 //! You can find out more about Gotham, including where to get help, at <https://gotham.rs>.
 //!
 //! We look forward to welcoming you into the Gotham community!
-#![doc(html_root_url = "https://docs.rs/gotham/0.5.0-rc.1")] // Update when changed in Cargo.toml
-#![warn(missing_docs, deprecated)]
+#![doc(html_root_url = "https://docs.rs/gotham/0.5.0-rc.1")]
+// Update when changed in Cargo.toml
```

Comment:
This was just me trying to remove the noise from my `cargo watch` output. Probably revert this and fix it in a separate PR.

Reply:
Should I revert these or remove these changes when rebasing the patchset? I'm going to rebase my part anyway, but not sure whether I should touch your commit too or leave it as-is.

Reply:
I'm happy for you to keep it as a separate commit or fixup my commit if you want. Up to you.